### PR TITLE
Reinforces airlocks in command areas 

### PR DESCRIPTION
**Pull request by Spess-Rando**
For those who believe that #35219 isn't enough.

### CODE REVIEW

**Comment by MMMiracles** [comment]
sTOP

**Comment by BeeSting12** [comment]
just reinforce caps office

**Comment by ShizCalev** [comment]
just as a note for folks who don't know what this does, this just delays hacking by 4 seconds by requiring you to welder the door first.

**Comment by epochayur** [comment]
captain's spare aside, there's no reason for this not to be a thing already

**Comment by ShizCalev** [comment]
tbh, should just set `security_level = AIRLOCK_SECURITY_METAL` on `/obj/machinery/door/airlock/command`, that way people don't accidentally use the editted door (which happens -alot-.)

**Comment by Slignerd** [comment]
Not all command airlocks were changed, only those adjacent to Captain's Office or Bridge, and on some maps, HoP's Office.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -6,22 +6,22 @@
...
 	new /obj/item/clothing/neck/petcollar(src)
 	new /obj/item/pet_carrier(src)
-	new /obj/item/storage/backpack/duffelbag/captain(src)
-	new /obj/item/clothing/head/crown/fancy(src)
-	new /obj/item/clothing/suit/captunic(src)
-	new /obj/item/clothing/under/captainparade(src)
-	new /obj/item/clothing/head/caphat/parade(src)
+	new /obj/item/clothing/shoes/sneakers/brown(src)
 	new /obj/item/clothing/under/rank/captain(src)
+	new /obj/item/clothing/suit/armor/vest/capcarapace
```
Comment (F-OS) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: no

**Comment by Teepeepee** [comment]
This is a nerf mostly to mothtide since they need more eye protection than other species can get in 20 seconds roundstart, bad change in my totally biased assistant moth oppinion.

**Comment by epochayur** [comment]
>balancing for non-humans

everyone point and laugh

**Comment by ExcessiveUseOfCobblestone** [comment]
YOU CAN ADD THIS YOURSELF INGAME JUST DO THAT OML

**Comment by Iatots** [comment]
But I want this as an alternative, not bundled with the other.

**Comment by 81Denton** [comment]
Just grab some metal, screwdriver the airlock and use it on that

Voila same result

**Comment by Slignerd** [comment]
@latots: Just look @ShizCalev's comment - it's not viable as an alternative at all.

**Comment by Slignerd** [comment]
@81Denton @ExcessiveUseOfCobblestone "Just do it ingame" absolutely fails to address the reason for which this PR was made - namely, the security of all access in the event of latejoining

**Comment by Slignerd** [comment]
@81Denton @ExcessiveUseOfCobblestone "Just do it ingame" absolutely fails to address the balance issue that this PR is aiming to fix.

**Comment by ExcessiveUseOfCobblestone** [comment]
I haven't been explained with any coherence the balance issue this is aiming to fix

**Comment by ShizCalev** [comment]
it really doesn't fix anything though since all it does is delay the time it takes to get to the id by 8 seconds.

**Comment by duncathan** [comment]
I don't believe that this is a necessary addition to the other pr; thank you for giving me the option

**Comment by Slignerd** [comment]
That's fine, the other PR was my main focus. 🙂

**Pull request closed without merging**